packages/amplify_core/lib/src/types/api/graphql/graphql_request.dart

```diff
@@ -19,17 +19,21 @@ import 'package:amplify_core/amplify_core.dart';
 class GraphQLRequest<T> {
   final String id = UUID.getUUID();
 
+  /// The body of the request, starting with the operation type and operation name.
+  ///
+  /// See https://graphql.org/learn/queries/#operation-name for examples and more information.
+  final String document;
+
   /// Only required if your backend has multiple GraphQL endpoints in the amplifyconfiguration.dart file. This parameter is then needed to specify which one to use for this request.
   final String? apiName;
 
+  /// Authorization type to use for this request. If not supplied, request will use
+  /// default endpoint mode. Note: this is an enum already defined in core.
```

Comment:
First line should be a single sentence (https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph).

I'm not sure that `Note: this is an enum already defined in core.` should be included. Is that intended to improve discoverability for the customer? Maybe we re-export export that enum api to make it more discoverable?

```suggestion
  /// Authorization type to use for this request.
  ///
  /// If not supplied, the request will use the default endpoint mode.
```

Reply:
good catch, thanks, this was bad copy pasta from design doc